docs/getting_oss_rules_of_play_rating.md

```diff
@@ -73,70 +78,65 @@ In addition, remember to keep the `OssRulesOfPlayRatingMarkdownFormatter.config.
 If everything is correctly set up, the output is going to look like the following:
 
 ```
-[+] The default value cache doesn't exist yet.
 [+] Okay, we have a GitHub token, let's try to use it
 [+] Let's gather info and calculate a rating for:
 [+]   https://github.com/SAP/code-pal-for-abap
-[+] Gathering info about project's README file ...
-[+] Cloning https://github.com/SAP/code-pal-for-abap ...
-[+] Fetching info about project's teams ...
-[+] Gathering info about vulnerability alerts ...
-[+] Gathering info about project's contributing guidelines ...
-[+] Gathering info about project's license ...
-[+] Figuring out how the project uses REUSE ...
-[+] Figuring out if the project has a security policy ...
```

Comment:
Please let these be as it is. Also is this the exact flow, which comes in CLI terminal?